db/migrate/20230503123612_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.text :title
+      t.text :text
+      t.date :createdat
+      t.date :updatedat
```

Comment:
- [optional] The createdat and updatedat columns are automatically created by rails using the `t.timestamps`. Kindly remove these please.